web/viewer.js

```diff
@@ -72,7 +72,6 @@ function getViewerConfiguration() {
       next: document.getElementById('next'),
       zoomIn: document.getElementById('zoomIn'),
       zoomOut: document.getElementById('zoomOut'),
-      viewFind: document.getElementById('viewFind'),
```

Comment:
This is still used in https://github.com/mozilla/pdf.js/blob/71d9f5f860a7ba3e7971c202fad78f8efe4f5b89/web/app.js#L1642 as far as I can tell, so won't this break?

Reply:
D'oh, that's obviously correct; thanks for spotting this obvious error on my part!

I'm removing the second commit, since it wasn't all that important anyway.